### PR TITLE
Create CQM QRDA home page

### PR DESCRIPTION
**Pull request by mattystank**
Worked on the user interface (UI) of the CQM (Clinical Quality Measures) page.

  - Added SVGs to the public CQM folder in the UI.
  - Implemented baseline UI for the CQM page with a grow effect, links, and text formatting.
  - Introduced the CQM component and a floating action button (FAB) to the CQM page, while deleting a placeholder.
  - Merged the 'dev' branch from the GitHub repository into the codebase.
  - Removed global components that had been relocated to the root directory.
  - Deleted unnecessary imports.
  - Eliminated the button base from the UI.
  - In the latest commit, commented out and removed the grow effect within elements, possibly indicating a refinement or modification in the UI implementation.

### CODE REVIEW

File: src/components/cqm-qrda/CqmHome.tsx [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+  justifyContent: 'flex-start',
+  textAlign: 'left',
+}
+
+const cypressCardImage = {
+  minWidth: '-webkit-fill-available',
+  height: '175px',
+}
+
+const flexiableContent = {
```
Comment (drbgfc) [inline src/components/cqm-qrda/CqmHome.tsx:66]: flexibleContent. Probabably a good idea to fix so no complile errors when used in the future when refactoring

File: src/components/cqm-qrda/CqmHome.tsx [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  justifyContent: 'space-between',
+  gap: 2,
+  p: 2,
+}
+
+const CqmHome = () => {
+  return (
+    <Box width={'100%'} pt={8}>
```
Comment (drbgfc) [inline src/components/cqm-qrda/CqmHome.tsx:77]: shouldn't need styles in Box since in root layout now, right? Just repeating?

**Review by drbgfc — COMMENTED** [review]
You did a lot of work here, nice!
I just scanned for now as it's late. I can review fully and merge in the morning.
Only thing I can see in a quick scroll is that it would be a 'nice to have' to break up CqmHome into a few more components as it's pretty large/a bit hard to work on fixes in a targeted manner as is possibly. You have it spelled out with comments, so could take some of those major sections and externalize.

Total side note, was wondering, do we intend for basically every component to have the comms fab? Even home?

**Comment by mattystank** [comment]
> Total side note, was wondering, do we intend for basically every component to have the comms fab? Even home?

Only the home screen & landing pages will have the FAB COM, once a user gets into a workflow I don't think FAB is important to them.

File: src/components/cqm-qrda/CqmHome.tsx [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                <Typography variant="body2" color="default">
+                  The Cypress Knowledge Base is a starting point for vendors and Authorized Testing Labs responsible for
+                  testing EHR technologies. The Knowledge Base has Frequently Asked Questions, as well as links to
+                  additional eCQM resources.
+                </Typography>
+              </CardContent>
+              <CardActions>
+                <Button
+                  disableRipple
+                  href="http://www.projectcypress.org/"
```
Comment (drbgfc) [inline src/components/cqm-qrda/CqmHome.tsx:116]: Shoudn't this be https?
https://projectcypress.org/

I can also update

File: src/components/cqm-qrda/CqmHome.tsx [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+            <VerticalCard
+              title={'Cypress Issue Tracker'}
+              description={'Track questions and issue that Cypress supports'}
+              buttonTitle={'TRACK your question'}
+              buttonIcon={<TrackChangesIcon />}
+              buttonHref={'https://jira.oncprojectracking.org/browse/CYPRESSef'}
+            />
+          </Box>
+        </Box>
+        <Divider sx={{ p: 2, borderBottomWidth: 2 }} />
```
Comment (drbgfc) [inline src/components/cqm-qrda/CqmHome.tsx:184]: I wonder if we should externalizer a specific divider with specific padding and width like this and just import and use that everywhere? Just a thought, not needing an update...

File: src/components/cqm-qrda/CqmHome.tsx [hunk 5]
```diff
@@ -0,0 +1,220 @@
...
+          display={'flex'}
+          flexDirection={'row'}
+          width={'100%'}
+          justifyContent={'space-between'}
+          alignItems={'stretch'}
+          gap={4}
+        >
+          {/* Cypress Card With Images */}
+          <Box display={'flex'} flexDirection={'row'} width={'60%'} gap={4}>
+            <Card
```
Comment (drbgfc) [inline src/components/cqm-qrda/CqmHome.tsx:92]: Interesting that you made this card from scratch. I wonder if we could have modified an existing SITE component and expanded with more parameters as needed?

File: src/components/cqm-qrda/VerticalCard.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import { Card, CardHeader, CardContent, Typography, Button } from '@mui/material'
+
+export interface VerticalCardProps {
```
Comment (drbgfc) [inline src/components/cqm-qrda/VerticalCard.tsx:3]: Awesome that this was added, and, I love how it supports Button Title

Comment (drbgfc) [reply src/components/cqm-qrda/VerticalCard.tsx:3]: It probably could have gone in the shared folder, though

Comment (drbgfc) [reply src/components/cqm-qrda/VerticalCard.tsx:3]: That way people will look there and see it for reuse

**Review by drbgfc — APPROVED** [review]
Approved and merging as I don't want to hold anyone up.
However, could you address the super minor 'fix' items in the comments with a future PR please? No rush. Thanks!

**Pull request merged**